examples/hosts/app-integration/schema-upgrade/src/containerKillBit.ts

```diff
@@ -76,24 +103,31 @@ export class ContainerKillBit extends DataObject implements IContainerKillBit {
     }
 
     protected async initializingFirstTime() {
+        const quorum = Quorum.create(this.runtime);
         const crc = ConsensusRegisterCollection.create(this.runtime);
         const taskManager = TaskManager.create(this.runtime);
+        this.root.set(quorumKey, quorum.handle);
         this.root.set(crcKey, crc.handle);
         this.root.set(taskManagerKey, taskManager.handle);
-        await Promise.all([
-            crc.write(markedForDestructionKey, false),
-            crc.write(deadKey, false),
-        ]);
+        quorum.set(markedForDestructionKey, false);
```

Comment:
No quorum events in the initialization path?

Reply:
It generally shouldn't be necessary since this should all run prior to attach, but I'll make sure -- the `queueMicrotask` calls might let us get out of order.

Looking further, I think I also have a problem for creating a new Quorum while already connected (since there will be members in the quorum).  I'll look at that more too.

Reply:
I added the event wait just to be certain -- probably isn't strictly necessary but fine for now.  I think I'll take a look at promise-based `set()` soon so this gets nicer looking then too.

Also fixed the issue I mentioned about creating a new Quorum in connected state in latest and added tests for it.